Settlement/Entry/Cancel.ts

```diff
@@ -1,35 +1,42 @@
+import { isoly } from "isoly"
 import { isly } from "isly"
 import { Amount } from "../../Amount"
 import { Authorization } from "../../Authorization"
+import { Identifier as SettlementIdentifier } from "../../Settlement/Identifier"
 import { Batch } from "../Batch"
 import { Fee } from "../Fee"
-
-export interface Cancel extends Cancel.Creatable {
+export interface Cancel extends Omit<Cancel.Creatable, "settlement"> {
 	status: "succeeded" | "failed"
 	reason?: string
+	created?: isoly.DateTime
+	settlement?: SettlementIdentifier
 }
 
 export namespace Cancel {
 	export interface Creatable {
 		type: "cancel"
 		authorization?: Authorization
-		reference: string
+		reference?: string
 		batch: Batch
 		fee?: Fee
 		amount?: Amount
+		settlement: SettlementIdentifier
```

Comment:
make it optional since older ones won't have it

Reply:
Neee, we decided to go with mandatory settlement on creatable cause there are no old createables, they get turned into proper Entries and they have optional settlement